engine/common/insight_log/insight_logs_enabled_check.py

```diff
@@ -1,15 +1,20 @@
-from django.apps import apps
+import logging
+
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
 
 
 def is_insight_logs_enabled(organization):
     """
     is_insight_logs_enabled checks if inside logs enabled for given organization.
+    Now it checks if oncall is deployed on same cluster that its grafana instance to be able to forward logs.
+    Or if it's Open Source :)
```

Comment:
> Or if it's Open Source :)

not clear, because this code is already opensource. Did you mean oncall deployed outside gcom from opesource code?

Reply:
I meant that this check return True if oncall is deployed as OSS

Reply:
this is nitpick, but the comment was not clear for me, worth to rewrite it